browser/js/global.js

```diff
@@ -391,6 +392,7 @@ class BrowserInitializer extends InitializerBase {
 		window.wasmEnabled = element.dataset.wasmEnabled.toLowerCase().trim() === "true";
 		window.indirectionUrl = element.dataset.indirectionUrl;
 		window.geolocationSetup = element.dataset.geolocationSetup.toLowerCase().trim() === "true";
+		window.slideshow = element.dataset.slideshowEnabled.toLowerCase().trim() === "true";
```

Comment:
can we follow the pattern: `slideshow` -> `slideshowEnabled`
or better `slideshow` -> `canvasSlideshowEnabled` or something similar to tell what exactly that option does (for someone unfamiliar)